.github/workflows/update_catalog.yml

```
@@ -0,0 +1,47 @@
+on:
+  push:
```

This should limit deploys to updates to the `catalog.json` file on the `master` branch:

```suggestion
  push:
    branches:
      - master
    paths:
      - "Catalog/catalog.json"
```

---

Just curious, why does this limit deploys?